filecoin-virtual-machine/fvm-contract-overview.md

```diff
@@ -6,12 +6,112 @@ description: >-
 
 # FVM Contract Overview
 
-### Lighthouse Contract Address
+## The Base and Full RaaS Interfaces
 
-**Calibration Testnet**: `0x6ec8722e6543fB5976a547434c8644b51e24785b`
+RaaS (renew, repair, replication) interacts with smart contracts to figure out which on-chain data deals to perform its services on. Specifically, it can interact with one of two types of interfaces:
 
-#### Functions for Developers (refer [RAAS starter kit](https://github.com/filecoin-project/raas-starter-kit) for more details), also check out the code examples for given functions [here](https://github.com/lighthouse-web3/raas-code-examples)
+- The Base Interface: A user deploys deploys a smart contract that inherits from [`IAggregatorOracle.sol`](https://github.com/xBalbinus/fevm-data-segment/blob/main/contracts/aggregator-oracle/IAggregatorOracle.sol) which allows them to rely on their own contract deployment to submit and complete data deals.
+- The Full Interface: A user relies on an existing FVM contract deployment to submit and complete data deals. Usually, the deployment is verified and maintained by storage infrastructures such as Lighthouse.
```

Comment:
Note: base and full interface naming is being removed. See more in thread here: https://filecoinproject.slack.com/archives/C0370V3JTPB/p1693420691727189?thread_ts=1693411733.707419&cid=C0370V3JTPB

Base interface is now called "self-hosted RaaS" (or in some cases "third party RaaS")
Full interface is now called "aggregator hosted RaaS"